examples/simulation/Tutorial1/src/FairTutorialDet1.cxx

```diff
@@ -24,40 +24,16 @@ FairTutorialDet1Geo* FairTutorialDet1::fgGeo = nullptr;
 
 FairTutorialDet1::FairTutorialDet1()
     : FairDetector("TutorialDet", kTRUE, kTutDet)
-    , fTrackID(-1)
-    , fVolumeID(-1)
-    , fPos()
-    , fMom()
-    , fTime(-1.)
-    , fLength(-1.)
-    , fELoss(-1)
-    , fIsInitialised(kFALSE)
     , fFairTutorialDet1PointCollection(new TClonesArray("FairTutorialDet1Point"))
```

Comment:
This initialization also can go to the the header file.

Reply:
Well, turned it over into a `unique_ptr` as well.